dev/appveyor-install-dependencies.ps1

```diff
@@ -120,8 +120,8 @@ $env:PATH = "$tools\sbt\bin;" + $env:PATH
 Pop-Location
 
 # ========================== Hadoop bin package
-# This must match the version at https://github.com/cdarlint/winutils/tree/master/hadoop-3.2.0
-$hadoopVer = "3.2.0"
+# This must match the version at https://github.com/cdarlint/winutils/tree/master/hadoop-3.3.5
+$hadoopVer = "3.3.5"
```

Comment:
Is it possible to use 3.3.6?

Reply:
Unfortunately, `3.3.6` currently does not exist on the URL: https://github.com/cdarlint/winutils/tree/master
<img width="902" alt="image" src="https://github.com/apache/spark/assets/15246973/a04b3285-27c2-4196-839b-668b55aab383">